README.md

```diff
@@ -14,6 +47,23 @@ To do that, we first do `npm run deployGovernance [mainnet|testnet]`. If the dep
 After the Timelock delay (currently 2 days), do `npm run timelockChangeAdminAndGovernorAcceptAdmin [mainnet|testnet]`. This script will read the config from `changeAdminConfig.json` to execute the transaction on Timelock. The GovernorAlpha contract will also accept the role of admin by calling `acceptAdmin()` in Timelock.
 
 
+### Deployed contract addresses 
+
+**Mainnet**
+| Contract       | Address                                    |
+| ---            | ---                                        |
+| VEX            |                                          |
+| Timelock       |                                          |
+| GovernorAlpha  |                                          |
+
+
+**Testnet**
+| Contract  | Address                                    |
+| ---       | ---                                        |
+| VEX      |  |
+| Timelock |  |
+| GovernorAlpha  |  |
+
```

Comment:
Can probably align these tables since we will need to add addresses a decent bit later

Reply:
will fix